solvers.tex

```diff
@@ -0,0 +1,127 @@
+%% Solvers in SymPy
+
+
+SymPy has a solvers module to solve equations symbolically. There are two
+submodules to solve algebraic equations in SymPy, referred to as old solve
```

Comment:
OK. Grammar problem remains, _solve_ is a verb, verbs cannot have adjectives or used as complements (unless -ing form is added). Maybe _old solve function_?